SayTheirNames/Resources/Base.lproj/Localizable.strings

```diff
@@ -33,12 +33,24 @@
 "person.news" = "News";
 "person.media" = "Media";
 "person.hashtags" = "Social Media Hashtags";
+
+// Get Involved - Slack
+"get_involved.title" = "HOW DO I GET INVOLVED";
+"get_involved.button" = "JOIN OUR SLACK CHANNEL";
+"get_involved.url" = "https://saytheirnames.slack.com/join/shared_invite/zt-eqjuatz7-fgh3zPRXIKiiXsC1Vf3oZA#/";
+
+// Get Involved - Developer
 "get_involved_dev.title" = "HOW DO I GET INVOLVED AS A DEVELOPER?";
-"get_involved_design.title" = "HOW DO I GET INVOLVED AS A DESIGNER?";
-"view_repo" = "VIEW OUR REPO";
-"repo_url" = "https://github.com/Say-Their-Name/say-their-names-ios";
-"join_slack" = "JOIN OUR SLACK CHANNEL";
-"slack_url" = "https://saytheirnames.slack.com/join/shared_invite/zt-eqjuatz7-fgh3zPRXIKiiXsC1Vf3oZA#/";
+"get_involved_dev.button" = "VIEW OUR REPO";
+"get_involved_dev.url" = "https://github.com/Say-Their-Name/say-their-names-ios";
```

Comment:
shouldn't this link to the org? not only iOS developers use iOS apps 😊 
the text then could be `VIEW OUR GITHUB` ?